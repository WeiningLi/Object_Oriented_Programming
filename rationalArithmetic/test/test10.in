create a 1/6
print a
create b 1/6
print b
+ a b
- b a
* a b
/ b a
+= a b
-= a b
create c 3/1
create d 1/3
* c d
/ c d
