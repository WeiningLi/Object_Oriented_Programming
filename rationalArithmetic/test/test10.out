1/6
1/6
1/3
0
1/36
1
1/3
1/6
1
9
1/0
1/0
1/0
1/0
